spec/system/candidate_entering_contact_details_spec.rb

```diff
@@ -38,7 +38,7 @@
     it 'sees an error summary with clickable links', js: true do
       expect(page).to have_content('There is a problem')
       click_on 'Enter your phone number'
-      expect(page).to have_selector('#contact_details_phone_number:focus')
+      expect(page).to have_selector('#contact-details-phone-number-field-error:focus')
```

Comment:
Does the field we're trying to check have a different class we can target it with?

This reads to me as 
> I expect my focus to be on the phone number field, which is marked as an error

whereas I think it's clearer to say 
> I expect my focus to be on the phone number field

Feels like the former is asserting two different things at once?

Reply:
Unfortunately if the form validation fails and the error summary is displayed then the form and form elements themselves are displayed with different ids see:

<img width="874" alt="Screenshot 2019-08-13 at 14 06 42" src="https://user-images.githubusercontent.com/47318392/62944087-dc1abe00-bdd3-11e9-971c-91c0767d35b2.png">

Not sure if there are any unique identifiers for the new field element that can be used on the test that don't include the `error` tag.